bin/ch-grow.py.in

```diff
@@ -465,6 +467,15 @@ class CLI_Print_Storage(ch.CLI_Action_Exit):
       print(namespace.storage)
       sys.exit(0)
 
+class CLI_List_Images(ch.CLI_Action_Exit):
+
+   def __call__(self, _, namespace, *args, **kwargs):
+      print("IMAGE") 
+      p = pathlib.Path(namespace.storage + '/img')
+      for child in p.iterdir():
```

Comment:
What's the argument for `pathlib` based solution here as opposed to `os.listdir()`?

Reply:
I initially used pathlib because I was intending to print metadata along with the image name. I think the pathlibapproach is generally more flexible, but it is a bit heavy for this current implementation.

Reply:
A quick test suggests `pathlib.Path.iterdir()` and `os.listdir()` give essentially equivalent results. If we stick with `pathlib`, probably we want to use idiomatic subpath appending code rather than appending strings. Finally, we don't use `pathlib` anywhere else. For these reasons I'd lean towards `os.listdir()`, and if there is a good argument to refactor using `pathlib`, put that into a separate PR.